driver/driver.go

```diff
@@ -53,10 +53,11 @@ func FromSource(source models.Source) (Driver, error) {
 			Credentials:      creds,
 			S3ForcePathStyle: aws.Bool(true),
 			MaxRetries:       aws.Int(maxRetries),
+			DisableSSL:       aws.Bool(source.DisableSSL),
 		}
 
 		if len(source.Endpoint) != 0 {
-			endpoint := fmt.Sprintf("https://%s", source.Endpoint)
+			endpoint := fmt.Sprintf("%s", source.Endpoint)
```

Comment:
Does this need the scheme in the source config now? That'll be backwards-incompatible.

Also this could just be `awsConfig.Endpoint = aws.String(source.Endpoint)`.